src/SmartComponents/SystemInventory/SystemInventoryHeader.js

```diff
@@ -75,92 +70,95 @@ const SystemInventoryHeader = ({
+                                            ) }
+                                            state="critical"
+                                            systemInventory
+                                        />
+                                    </InsightsLink>
+                                    }
+                                </FlexItem>
+                                {inventoryTotalFetchStatus === 'rejected' &&
+                                    <FailState appName='Inventory' isSmall />
                                 }
+                            </Flex>
+                            <FlexItem align={{ md: 'alignRight' }}>
+                                <Link to="/settings/integrations">
+                                    <Button
+                                        className='pf-v5-u-mr-sm pf-v5-u-font-size-md'
+                                        variant='secondary'
+                                        size='sm'
+                                    >
+                                        { intl.formatMessage(messages.configureIntegrations) }
+                                    </Button>
+                                </Link>
+                                <InsightsLink app='registration' to="/">
+                                    <Button
+                                        variant='primary'
+                                    >
+                                        { intl.formatMessage(messages.systemInventoryCTA) }
+                                    </Button>
+                                </InsightsLink>
                             </FlexItem>
-                            {inventoryTotalFetchStatus === 'rejected' &&
-                                <FailState appName='Inventory' isSmall />
-                            }
                         </Flex>
-                        <FlexItem align={{ md: 'alignRight' }}>
-                            <Link to="/settings/integrations">
-                                <Button
-                                    className='pf-v5-u-mr-sm pf-v5-u-font-size-md'
-                                    variant='secondary'
-                                    size='sm'
-                                >
-                                    { intl.formatMessage(messages.configureIntegrations) }
-                                </Button>
-                            </Link>
-                            <InsightsLink app='registration' to="/">
-                                <Button
-                                    variant='primary'
-                                >
-                                    { intl.formatMessage(messages.systemInventoryCTA) }
-                                </Button>
-                            </InsightsLink>
-                        </FlexItem>
                     </Flex>
-                </Flex>
+                    <EdgeDevicesWarning />
```

Comment:
Looking at the Vulnerability Systems page, which is taken as an example for this message, I think the warning should rather be in the content area and not the header part of the page.

Reply:
according to the discussion on the ticket: https://issues.redhat.com/browse/RHINENG-2743 the UX placed the banner inside, because it only affects the header content, not the whole page
it does not look the best, that is true, but i don't see any other solution which would communicate the same thing

Reply:
@xmicha82 My 5 cents regarding alert placement, According to the mock alert should be placed above inventory data 

Currently: 
![image](https://github.com/RedHatInsights/insights-dashboard/assets/33912805/083f88aa-8828-4786-b356-f0ae672c33fc)

Expected:
![image](https://github.com/RedHatInsights/insights-dashboard/assets/33912805/aac51ec0-c62c-41d4-bf11-f38b0794ae1d)

Reply:
yeah, i know about this, i was just waiting for the resolution from Andy Burka on where to place it

Reply:
@xmicha82 Last piece is the message itself - should be `"Immutable (OSTree) systems are included in the inventory totals."` instead of `"Immutable (OSTree) systems data is not included in the dashboard."`